bmcd/src/main.rs

```diff
@@ -0,0 +1,51 @@
+use actix_files::Files;
+use actix_web::{middleware, web::Data, App, HttpServer};
+use log::LevelFilter;
+use tokio::sync::Mutex;
+use tpi_rs::app::bmc_application::BmcApplication;
+
+mod legacy;
+
+#[actix_web::main]
+async fn main() -> anyhow::Result<()> {
+    init_logger();
+
+    let bmc = Mutex::new(BmcApplication::new().await?);
```

Comment:
The `BmcApplication` is already `Sync` as we handle thread sychronization inside of it. So no need to wrap it with a mutex.
```suggestion
    let bmc = Data::new(BmcApplication::new().await?);
```